inc/template-tags.php

```diff
@@ -92,51 +131,99 @@ function primer_the_site_title() {
  */
 function primer_the_site_description() {
 
-	$html = sprintf(
-		'<div class="site-description">%s</div>',
-		get_bloginfo( 'description' )
-	);
-
 	/**
-	 * Filter the site description HTML.
+	 * Filter the site description display args.
 	 *
-	 * @since 1.0.0
+	 * @since NEXT
 	 *
-	 * @var string
+	 * @var array
 	 */
-	echo (string) apply_filters( 'primer_the_site_description', $html ); // xss ok.
```

Comment:
@EvanHerman We should probably deprecate this filter in favor of `primer_the_site_description_args`.